src/SoftenedLinearPrior.cc

```diff
@@ -93,8 +92,8 @@ SoftenedLinearPrior::SoftenedLinearPrior(Control const & ctrl) :
             "ellipticityMaxOuter must be > 0"
         );
     }
-    pex::logging::Debug log("meas.modelfit.SoftenedLinearPrior");
-    log.debug<7>("Constructing new SoftenedLinearPrior");
+    LOG_LOGGER trace3Logger = LOG_GET("TRACE3:meas.modelfit.SoftenedLinearPrior");
```

Comment:
I'm not sure where my code developed this ':' instead of the '.'  I will fix it.